### ISSUE TITLE
[pointer_interceptor] Wrapped elements are not accessible.

### ISSUE THREAD

**Issue by ditman** [flutter/flutter#100157]
Internal: b/227167830

When a Button is wrapped by PointerInterceptor, it becomes not accessible (TalkBack doesn't read that there's a button).

Maybe the empty div on the pointer interceptor needs `aria-hidden="true"`?

Note: this works for desktop web, just not Android/iOS.

## Steps to Reproduce

1. Create a default flutter app
2. Check that talkback works on the FaB
3. Wrap the FaB in a PointerInterceptor
4. Check that talkback does NOT work on the FaB.

**Expected results:**

* I expect the button to be read by the screen-reader.

**Actual results:**

* The button is not found by the screen-reader.

_(Reported internally)_

**Comment by yjbanov** [comment]
This is about to block a customer launch in a couple of weeks, so raising priority to P2.

**Comment by yjbanov** [comment]
Update: the issue is that we put `<flt-semantics-host>` behind `<flt-scene-host>` and since platform views are embedded into `<flt-scene-host>` they take precedence in hit test order. In the example above, the pointer interceptor sits on top of the accessibility tree of the button, and even though the button is reachable using UI traversal shortcuts, it is not reachable using the finger drag gesture.

The fix is not to simply change the order. If we do that for all apps, we'll end up with the opposite problem, i.e. the accessibility tree of the platform views will become obscured by Flutter's accessibility tree. The [scenelet design](https://flutter.dev/go/flutter-web-scenelets) attempts to solve it more fully, but we haven't gotten to those parts yet.

A short-term solution might be to give the developer control over the relative stacking order of `<flt-semantics-host>` and `<flt-scene-host>`. This way the developer would be able to adjust to the situation at least on a coarse-grained level. For example, the app could mostly keep platform views behind the scene host, then in some cases, where an interactive platform view is in the foreground, the app could dynamically switch the order.

**Comment by yjbanov** [comment]
Update: I'm testing a fix that moves the semantics tree on top of the scene host, but allows "holes" such that platform views that appear in the background can still be reached.
